packages/design-system/src/components/Navigations/Pagination/PageBar.tsx

```diff
@@ -0,0 +1,147 @@
+import styled from '@emotion/styled';
+import classNames from 'classnames';
+import { colors } from '../../../styles';
+
+import Icon from '../../Base/Icon/Icon';
+import usePagination, { TILDE } from './usePagination';
+
+export interface PageBarProps {
+  onPageChange: (page: number) => void;
+  totalCount: number;
+  siblingCount?: number;
+  currentPage: number;
+  pageSize: number;
+  className?: string;
+}
+export default function PageBar({
+  onPageChange,
+  totalCount,
+  siblingCount = 2,
+  currentPage,
+  pageSize,
+  className,
+}: PageBarProps) {
+  const paginationRange: (string | number)[] = usePagination({
+    currentPage,
+    totalCount,
+    siblingCount,
+    pageSize,
+  });
+
+  const lastPage = paginationRange[paginationRange.length - 1];
+
+  const PageContainer = styled('div')`
+    font-size: 24px;
+    height: 24px;
+    display: flex;
+  `;
+
+  type PageButtonProps = {
+    isSelected: boolean;
+  };
+
+  const ArrowButton = styled('button')`
+    width: ${Math.ceil(totalCount / pageSize) < 100 ? '48px' : '64px'};
+    height: ${Math.ceil(totalCount / pageSize) < 100 ? '48px' : '64px'};
+    background-color: ${colors.grey3};
+    padding: 5px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  `;
+
+  const PageButton = styled('button')`
+    width: ${Math.ceil(totalCount / pageSize) < 100 ? '48px' : '64px'};
+    height: ${Math.ceil(totalCount / pageSize) < 100 ? '48px' : '64px'};
+    padding: 5px;
+    border: 1px solid ${colors.grey4};
+    color: ${(props: PageButtonProps) => !props.isSelected && colors.grey4};
+    justify-content: center;
+    align-items: center;
+  `;
+
+  const onPrev = () => {
+    onPageChange(currentPage - 1);
+  };
+
+  const onNext = () => {
+    onPageChange(currentPage + 1);
+  };
+
+  if (currentPage === 0 || paginationRange.length < 2) {
+    return (
+      <PageContainer className={classNames('hermes-pagination-container')}>
```

Comment:
prop으로 받는 className을 이 요소에 바인딩하려면 다음과 같이 사용해야 합니다.
```suggestion
      <PageContainer className={classNames('hermes-pagination-container', className)}>
```
그 외에 아래 요소들은 클래스명이 하나여서 classNames로 묶을 필요 없을 것 같아요.